### PR TITLE
Adds a guide on using Packer with CI/CD

### PR DESCRIPTION
**Pull request by skierkowski**
This PR adds a new guide to https://www.packer.io/guides/index.html for using Packer in CI/CD, pushing changes to an artifact store, and integrating with Terraform Enterprise.

### CODE REVIEW

File: website/source/guides/packer-on-cicd/building-image-in-cicd.html.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+page_title: Building Images in CI/CD
+description: |-
+  ...
+---
+
+# Building Images in CI/CD
+
+The following guides from our amazing partners show how to use their service to build images with Packer.
+
+- [How to Build Immutable Infrastructure with Packer and CircleCI Workflows](#)
```
Comment (nfagerlund) [inline website/source/guides/packer-on-cicd/building-image-in-cicd.html.md]: What's this link supposed to be?

**Comment by mwhooker** [comment]
Could you please move the images into `assets/images/guides`?

File: website/source/guides/packer-on-cicd/building-image-in-cicd.html.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-build-image
+page_title: Building Images in CI/CD
+description: |-
```
Comment (nfagerlund) [inline website/source/guides/packer-on-cicd/building-image-in-cicd.html.md]: Let's delete this if we're not using it.

File: website/source/guides/packer-on-cicd/building-virtualbox-image.html.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+page_title: Building a VirtualBox Image with Packer in TeamCity
+description: |-
+  ...
+---
+
+# Building a VirtualBox Image with Packer in TeamCity
+
+This guide walks through the process of building a VirtualBox image using Packer on a new TeamCity Agent. Before getting started you should have access to a TeamCity Server.
+
+The Packer VirtualBox builder requires access to VirtualBox which should run on a bare-metal machine as virtual machines should not run inside other virtual machines. This is also true for the [VMWare](https://www.packer.io/docs/builders/vmware.html) and the [QEMU](https://www.packer.io/docs/builders/qemu.html) Packer builders.
```
Comment (mwhooker) [inline website/source/guides/packer-on-cicd/building-virtualbox-image.html.md]: > as virtual machines should not run inside other virtual machines

I think it's more accurate to say that virtualization is generally not supported on cloud instances.

Maybe

> The Packer VirtualBox builder requires access to VirtualBox, which generally needs to run on a bare-metal machine.

File: website/source/guides/packer-on-cicd/building-virtualbox-image.html.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  facility         = "ams1"
+  operating_system = "ubuntu_16_04"
+  billing_cycle    = "hourly"
+  project_id       = "${packet_project.teamcity_project.id}"
+}
+```
+
+## 2. Install VirtualBox and TeamCity Dependencies
+
+VirtualBox must be installed on the new instance along and TeamCity requires the JDK prior to installation. This guide uses Ubuntu as the Linux distribution, so you may need to adjust these commands for your distribution of choice.
```
Comment (mwhooker) [inline website/source/guides/packer-on-cicd/building-virtualbox-image.html.md]: missing word?

File: website/source/guides/packer-on-cicd/building-virtualbox-image.html.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+Add one **Build Step: Command Line** to the build.
+
+![TeamCity screenshot: New Build](./images/teamcity_new_build.png)
+
+In the **Script content** field add the following:
+
+```shell
+#!/usr/bin/env bash
+/root/packer build -only=virtualbox-iso -var "headless=true" ./packer.json
```
Comment (mwhooker) [inline website/source/guides/packer-on-cicd/building-virtualbox-image.html.md]: headless=true is only applicable if you've structured your packer.json just so. I recommend we use an example json somewhere

File: website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# Uploading Images to Artifact Stores
+
+Once the image is generated it will be used by other parts of your operations workflow. For example, it is common to build VirtualBoxes with Packer to be used as base boxes in Vagrant.
+
+On the agent machine install the [AWS Command Line Tool](https://aws.amazon.com/cli/). Since this is a one-time operation, this can be incorporated into the initial provisioning step when installing other dependencies.
+
+```shell
+pip install awscli
```
Comment (mwhooker) [inline website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md]: worth calling out the requirements from http://docs.aws.amazon.com/cli/latest/userguide/installing.html ?

```
Requirements

Python 2 version 2.6.5+ or Python 3 version 3.3+
Windows, Linux, macOS, or Unix
```

File: website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+layout: guides
+sidebar_current: guides-packer-on-cicd-upload-image-to-artifact-store
+page_title: Uploading Images to Artifact Stores
+description: |-
+  ...
+---
+
+# Uploading Images to Artifact Stores
+
+Once the image is generated it will be used by other parts of your operations workflow. For example, it is common to build VirtualBoxes with Packer to be used as base boxes in Vagrant.
```
Comment (mwhooker) [inline website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md]: VirtualBoxes -> VirtualBox images

File: website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-upload-image-to-artifact-store
+page_title: Uploading Images to Artifact Stores
+description: |-
+  ...
+---
+
+# Uploading Images to Artifact Stores
```
Comment (nfagerlund) [inline website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md]: The title sounds like this is meant to be a more generic guide, but it seems to be TeamCity and S3-specific.

File: website/source/guides/packer-on-cicd/build-virtualbox-image.html.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+# Build a VirtualBox Image with Packer in TeamCity
+
+This guide walks through the process of building a VirtualBox image using
+Packer on a new TeamCity Agent. Before getting started you should have access
+to a TeamCity Server.
+
+The Packer VirtualBox builder requires access to VirtualBox, which needs to run
+on a bare-metal machine, as virtualization is generally not supported on cloud
+instances. This is also true for the [VMWare](/docs/builders/vmware.html) and
+the [QEMU](/docs/builders/qemu.html) Packer builders.
```
Comment (rickard-von-essen) [inline website/source/guides/packer-on-cicd/build-virtualbox-image.html.md]: A bit late on the ball here but both GCP and Azure support nested virtualization. I think this should be rephrased to be accurate.

File: website/source/guides/packer-on-cicd/build-virtualbox-image.html.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+
+We will use Chef's [Bento boxes](https://github.com/chef/bento) to provision an
+Ubuntu image on VirtualBox. We will use a fork of this repository as the
+project we will build.
+
+## 1. Provision a Bare-metal Machine
+
+The Packer VirtualBox builder requires running on bare-metal (hardware). If you
```
Comment (rickard-von-essen) [inline website/source/guides/packer-on-cicd/build-virtualbox-image.html.md]: And here.

**Pull request merged**